add new methods (#1031)
rsksmart/devportal: content/rsk-devportal/tools/rpc-api.md

        "result": "0x4bdcfb"
    }
    ```
- eth_call

>     - Executes a new message call immediately without creating a transaction on the block chain.

Reviewer: ```suggestion
    - Executes a new message call immediately without creating a transaction on the blockchain.
```

    - _Params:_ 
        - `transaction`: object, the transaction call object which contains the following fields:
            - **from:** String, the address from which the transaction is sent
            - **to:** String, required, the address to which the transaction is addressed
            - **gas:** String, the integer of gas provided for the transaction execution

>             - **gasPrice:** String, the integer of gasPrice used for each paid gas encoded as hexadecimal

Reviewer: ```suggestion
            - **gasPrice:** String, the integer of the `gasPrice` used for each paid gas, encoded as a hexadecimal
```

            - **value:** String, the integer of value sent with this transaction encoded as hexadecimal

>             - **data:** string, the hash of the method signature and encoded parameters. For more information, see the Contract ABI description in the Solidity documentation

Reviewer: Please add link to contract ABI specification

Author: done

        - `blockNumber`: String, required. The number of the block (in hex) from which the number of transactions is required, OR one of the following block tags:
        -   **latest:** the most recent block the client has available.
        -   **earliest:** the lowest numbered block the client has available.

>         -   **pending:** A sample next block built by the client on top of latest and containing the set of transactions usually taken from local mempool. Intuitively, you can think of these as blocks that have not been mined yet. 

Reviewer: ```suggestion
        -   **pending:** A sample next block built by the client on top of latest and containing the set of transactions usually taken from a local mempool. Intuitively, you can think of these as blocks that have not been mined yet. 
```

        --header 'accept: application/json' \
        --header 'Content-Type: application/json'  \
        --data '{
            "jsonrpc":"2.0",
            "method":"eth_chainId",
            "params":[],
            "id":0
        }'
        ```
    - **Example Response:**
        ```shell
        {
            "jsonrpc": "2.0",
            "id": 0,
            "result": "0x1f"
        }
        ```
- eth_estimateGas
    - _Method:_
        - Generates and returns an estimate of how much gas is necessary to allow the transaction to complete. The transaction will not be added to the blockchain. 
    - _Params:_
        - **transaction:** object, the transaction call object which contains the following fields:
            - **from:** String, the address from which the transaction is sent
            - **to:** String, required, the address to which the transaction is addressed
            - **gas:** String, the integer of gas provided for the transaction execution
            - `gasPrice`: String, the integer of gasPrice used for each paid gas encoded as hexadecimal
            - `value`: String, the integer of value sent with this transaction encoded as hexadecimal

>             - `data`: string, the hash of the method signature and encoded parameters. For more information, see the Contract ABI description in the Solidity documentation

Reviewer: Same here

Author: done

        - `blockNumber`: String, optional. The number of the block (in hex) from which the number of transactions is required, OR one of the following block tags:
            - **latest:** the most recent block the client has available.
            - **earliest:** the lowest numbered block the client has available.

>             - **pending:** A sample next block built by the client on top of latest and containing the set of transactions usually taken from local mempool. Intuitively, you can think of these as blocks that have not been mined yet. 

Reviewer: ```suggestion
            - **pending:** A sample next block built by the client on top of latest and containing the set of transactions usually taken from a local mempool. Intuitively, you can think of these as blocks that have not been mined yet. 
```

        - Returns information about a block by blockNumber.
    - _Params:_
        - Block: String: required, either the hexadecimal value of a blockNumber, OR one of the following block tags:
            - latest: the most recent block the client has available.
            - earliest: the lowest numbered block the client has available.

>             - pending: A sample next block built by the client on top of latest and containing the set of transactions usually taken from local mempool. Intuitively, you can think of these as blocks that have not been mined yet.

Reviewer: ```suggestion
            - pending: A sample next block built by the client on top of latest and containing the set of transactions usually taken from a local mempool. Intuitively, you can think of these as blocks that have not been mined yet.
```

        - Option: Boolean, optional. 
            - false: returns only the hashes of the transactions (default)

>             - true: returns the full transactions objects

Reviewer: ```suggestion
            - true: returns the full transactions object
```

    - _Params:_
        - Address: String: required, address
        - Block: String, required, either the hexadecimal value of a blockNumber, OR a blockHash, OR one of the following block tags:
            - latest: the most recent block the client has available.
            - earliest: the lowest numbered block the client has available.

>             - pending: A sample next block built by the client on top of latest and containing the set of transactions usually taken from local mempool. Intuitively, you can think of these as blocks that have not been mined yet. 

Reviewer: ```suggestion
            - pending: A sample next block built by the client on top of latest and containing the set of transactions usually taken from a local mempool. Intuitively, you can think of these as blocks that have not been mined yet. 
```

                }, {...}
            ] }
        ```
- eth_getStorageAt
    - _Method:_ `eth_getStorageAt`
        - Returns the value from a storage position at a given address.
    - _Params:_
        - **Address:** String, required - A string representing the address (20 bytes) of the storage.
        - **Position:** String, required - A hexadecimal code of the position in the storage.
        - **Block:** String: required, either the hexadecimal value of a **blockNumber**, OR a blockHash, OR one of the following block tags:
            - **Latest:** the most recent block the client has available.
            - **Earliest:** the lowest numbered block the client has available.

>             - **Pending:** A sample next block built by the client on top of latest and containing the set of transactions usually taken from local mempool. Intuitively, you can think of these as blocks that have not been mined yet. 

Reviewer: ```suggestion
            - **Pending:** A sample next block built by the client on top of latest and containing the set of transactions usually taken from a local mempool. Intuitively, you can think of these as blocks that have not been mined yet. 
```

        {
            "jsonrpc": "2.0",
            "id": 0,
            "result": "0x3e"
        }
        ```
- eth_sendRawTransaction
    - _Method:_ `eth_sendRawTransaction`

>         - Creates new message call transaction or a contract creation for signed transactions.

Reviewer: ```suggestion
        - Creates a new message call transaction or a contract creation for signed transactions.
```
